### PR TITLE
fix: vue3-flicking can't refresh panel element

### LINKED ISSUE #585: Fade plugin won't work with renderOnlyVisible [vue3-flicking]
## Description
Hi,
Fade plugin tries to modify the original panel style but the diffResult won't carry them so the `opacity` was never actually set for Vue elements.

## Steps to check or reproduce
Sorry for missing the repo. I'm in the middle of something. Will provide soon if it's not clear for you.

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#585

## Details
This fixes #585

### CODE REVIEW

File: packages/vue3-flicking/src/VueElementProvider.ts [hunk 1]
```diff
@@ -8,9 +8,20 @@ import VuePanel from "./VuePanel";
...
 
-  public get element() { return this._el.nativeElement; }
   public get rendered() { return !this._el.hide; }
+  public get element() {
+    const el = this._el.$el.nextSibling as HTMLElement;
+
+    if (!el || el.nodeType !== Node.ELEMENT_NODE) {
+      return this._cachedElement;
+    } else {
+      this._cachedElement = el;
```
Comment (mixed) [inline packages/vue3-flicking/src/VueElementProvider.ts:20]: How about it?
```js
if (el && el.nodeType === Node.ELEMENT_NODE) {
    this._cachedElement = el;
}

return this._cachedElement;
```

**Review by mixed — APPROVED** [review]
LGTM

File: packages/vue3-flicking/src/VueElementProvider.ts (hunk 1 above)
Comment (WoodNeck) [reply packages/vue3-flicking/src/VueElementProvider.ts:20]: @mixed 
That looks much better 👍

File: packages/vue3-flicking/demo/App.vue [hunk 2]
```diff
@@ -71,16 +71,12 @@
           <span class="flicking-arrow-next"></span>
         </template>
       </Flicking>
-      <Flicking :plugins="arrow">
+      <Flicking :plugins="fade" :options="{ renderOnlyVisible: true }">
```
Comment (daybrush) [inline packages/vue3-flicking/demo/App.vue:74]: Can't the demo use both fade and arrow?

**Review by daybrush — APPROVED** [review]
LGTM

File: packages/vue3-flicking/demo/App.vue (hunk 2 above)
Comment (WoodNeck) [reply packages/vue3-flicking/demo/App.vue:74]: It can, but I just separated it to test the combination of fade + renderOnlyVisible

**Pull request merged**